client/state/imports/utils.js

```diff
@@ -57,11 +56,7 @@ function replaceUserInfoWithIds( customData ) {
 	return Object.assign( {}, customData, {
 		sourceAuthors: customData.sourceAuthors.map(
 			author =>
-				author.mappedTo
-					? Object.assign( {}, author, {
-							mappedTo: author.mappedTo.ID,
-						} )
-					: author
+				author.mappedTo ? Object.assign( {}, author, { mappedTo: author.mappedTo.ID } ) : author
```

Comment:
feel free to use shorter notation - no need to be verbose here

```js
author.mappedTo ? { ...author, mappedTo: author.mappedTo.ID } : author
```

Reply:
👍 will make that change